src/hardware/serialport/nullmodem.h

```diff
@@ -51,7 +51,7 @@ class CNullModem final : public CSerial {
 	void setDTR(bool val) override;
 	void handleUpperEvent(uint16_t type) override;
 
-	SocketTypesE socketType = SOCKET_TYPE_TCP;
+	SocketType socketType = SocketType::Tcp;
```

Comment:
nice! Can the `SOCKET_TYPE_TCP` and `SOCKET_TYPE_ENET` definitions be eliminated (where ever they are?)

Reply:
They are now gone,transformed into `SocketType::Tcp` and `SocketType::Enet`.